src/web/resources/report.js

```diff
@@ -35,6 +35,49 @@ function Element(tagname, props, children) {
     return $elt;
 }
 
+const Filters = new Component("#filters", {
+    setup: function () {
+        const filterByExStart = Element("a", "Remove \"ex-start\" ")
+        const filterByBadCases = Element("a", "View only \"bad\" cases ")
+        const reset = Element("a", "Rest Filters")
+
+        const filters = Element("div", [
+            Element("div", "Filters"),
+            Element("div", [
+                filterByExStart,
+                filterByBadCases,
+                reset])])
+
+        reset.addEventListener("click", function() {
+            location.reload()
+        })
+
+        filterByExStart.addEventListener("click", function () {
+            const parent = document.querySelector("#results tbody")
+            const siblingsList = document.querySelectorAll("#results tbody tr")
+            siblingsList.forEach((child, n, p) => {
+                if (child.classList.contains("ex-start")) {
+                    parent.removeChild(child)
```

Comment:
1. You can just call `remove`
2. Don't actually remove the row, just set `child.style.display = "none"` if you want it to disappear and set `child.style.display = "table-row"` if you want it to reappear. That keeps it on the page and just shows / hides it. That'll also let you implement reset in a normal way.
3. You might want to generalize a bit and have a helper function that iterates over rows and shows/hides them based on a function that tells you whether or not you want that row.